proto/cws/dumpsv1/activity_dump.proto

```diff
@@ -50,7 +50,6 @@ message ProcessActivityNode {
     ProcessInfo process = 1;
     reserved 2;
     GenerationType generation_type = 8;
-    bool is_exec_child = 10;
```

Comment:
If this is removed from this message, should we add 10 to `reserved` list?

Reply:
Yes it should, but we've introduced this field for the 7.47 agent release and weren't using it before. So we're not breaking anything by removing it entirely because it wasn't used 👍🏻